File: 프로그래머스/lv0/120907. OX퀴즈/OX퀴즈.js
function solution(quiz) {
  var answer = [];
  quiz.forEach((item) => {
    const quizToList = item.split(" ");
    // 부호가 '+'인 경우
    if (quizToList[1] === '+') {
      if (Number(quizToList[0]) + Number(quizToList[2]) === Number(quizToList[4])) {
        answer.push('O');
      } else {
        answer.push('X');
      }
    // 부호가 '-'인 경우
    } else {
      if (Number(quizToList[0]) - Number(quizToList[2]) === Number(quizToList[4])) {
        answer.push('O');
      } else {
        answer.push('X');
      }
    }
  })
  return answer;
}

File: 프로그래머스/lv0/120907. OX퀴즈/OX퀴즈.py
def solution(quiz):
    result = []
    
    for item in quiz:
        item = item.split()
        answer = int(item[0])
        for i in range(1, len(item)):
            if item[i] == "-":
                answer -= int(item[i + 1])
                i += 1
            elif item[i] == "+":
                answer += int(item[i + 1])
            elif item[i] == "=":
                if answer == int(item[i + 1]):
                    result.append("O")
                else:
                    result.append("X")
            
    return result